src/pepr/operator/controllers/exemptions/exemptions.ts

```diff
@@ -0,0 +1,110 @@
+import { Log } from "pepr";
+import { policies } from "../../../policies/index";
+import { Matcher, Policy, UDSExemption } from "../../crd";
+
+// *** Using setItemAndWait() ***
+// Add Exemptions to Pepr store as "policy": "[{matcher}]"
+// export async function addExemptions(exmpt: UDSExemption) {
+//   const t0 = performance.now();
+//   const { Store } = policies;
+//   if (exmpt.spec && exmpt.spec.exemptions) {
+//     for (const e of exmpt.spec.exemptions) {
+//       const name = removeRegexSlash(e.matcher.name);
+//       for (const p of e.policies) {
+//         const exemptionList = JSON.parse(Store.getItem(p) || "[]");
+//         exemptionList.push({ namespace: e.matcher.namespace, name: name });
+//         await Store.setItemAndWait(p, JSON.stringify(exemptionList));
+//       }
+//     }
+//   }
+//   const t1 = performance.now();
+//   Log.debug(`Time to complete exemption write: ${t1 - t0}`);
+// }
+
+// Remove leading and trailing '/' if added by user to matcher name
+function removeRegexSlash(name: string) {
+  if (name[0] === "/" && name[name.length - 1] === "/") {
+    name = name.slice(1, name.length - 1);
+  }
+  return name;
+}
+
+function isAlreadyAdded(matchers: Matcher[], name: string) {
+  for (const m of matchers) {
+    if (m.name === name) {
+      return true;
+    }
+  }
+}
+
+const policyList = Object.values(Policy);
+
+// *** Use Local Map to then Update Store ***
+// Add Exemptions to Pepr store as "policy": "[{matcher}]"
+export function processExemptions(exmpt: UDSExemption) {
```

Comment:
Won't comment on all of them, but lots of places where `exempt` is missing the second `e` like this. Should be an easy find/replace.

Reply:
i purposely left it out i guess as a shorter abbreviation, but yeah that seems kind of stupid now. I'll change it.

Reply:
`xempt` is the way to go 😛, but yeah I think full name is better so that anyone new to the code isn't guessing at the right vars to pass around/use. Especially when we have exemption and others fully written out.